src/psyclone/tests/psyGen_test.py

```diff
@@ -3258,74 +3258,82 @@ def test_symbol_scalar_array():
     assert not sym2.is_scalar
     assert sym2.is_array
 
+    sym3 = Symbol("s3", "real",
+                  interface=Symbol.FortranGlobal(module_use="my_mod"))
+    assert ("s3: <real, Scalar, global=FortranModule(my_mod)"
+            in str(sym3))
 
-def test_symbol_is_input_setter():
-    '''Test that a Symbol's is_input method can be set if given a new
-    valid value, otherwise it raises a relevant exception.'''
+    sym2._shape.append('invalid')
+    with pytest.raises(InternalError) as error:
+        _ = str(sym2)
+    assert ("Symbol shape list elements can only be 'Symbol', 'integer' or "
+            "'None', but found") in str(error.value)
```

Comment:
This last 10 lines did not merge as expected, they belong to the `test_symbol_can_be_printed()` test

Reply:
Oops again. Fixed.